frontend/src/components/Calendar.tsx

```diff
@@ -226,50 +233,50 @@ const CalContainer = styled.div`
   }
 `
 
-interface IUseCurrentWeekState {
-  readonly weekStartDate: Date
-  readonly isTouched: boolean
+// pull the week from the URL otherwise default to the current time.
+function getToday(search: string): Date {
+  const week = queryString.parse(search).week
+  if (week == null || Array.isArray(week)) {
+    return new Date()
+  }
+  const parsedDate = parseISO(week)
+  if (isValid(parsedDate)) {
+    return parsedDate
+  }
+  return new Date()
 }
 
 function useCurrentWeek() {
-  const today = useCurrentDay()
```

Comment:
removed updating the current day via a timer as it was complicating things and isn't super useful.